app/com/gu/contentapi/sanity/EditorsPicksContainsItemsTest.scala

```diff
@@ -12,9 +12,7 @@ class EditorsPicksContainsItemsTest(context: Context) extends SanityTestBase(con
     whenReady(httpRequest) { result =>
       assume(result.status == 200, "Service is down")
       val json = Json.parse(result.body)
-      val editorsPicksJson = (json \ "response" \ "editorsPicks")
-      val editorsPicksList = editorsPicksJson.as[List[Map[String, String]]]
-      editorsPicksList should not be empty
+      (json \ "response" \ "editorsPicks").toOption should not be(None)
```

Comment:
Don't we want to check that it's not an empty list, i.e. there is at least one editor's pick? e.g.

```scala
(json \ "response" \ "editorsPicks").as[List[Any]] should not be empty
```